cfgov/jinja2/v1/landing-page/index.html

```diff
@@ -0,0 +1,161 @@
+
+{% extends 'layout-2-1-bleedbar.html' %}
+
+{# Import organisms and molecules used in the template. #}
+{% import 'molecules/hero.html' as hero %}
+{% import 'molecules/half-width-link-blob.html' as link_blob %}
+{% import 'molecules/image-text-25-75.html' as image_text_25_75 %}
+{% import 'molecules/call-to-action.html' as call_to_action %}
+{% import 'organisms/well.html' as well %}
+{% import 'organisms/email-signup.html' as email_signup %}
+{% import 'molecules/related-posts.html' as related_posts %}
+{% import 'molecules/text-introduction.html' as text_introduction %}
+{% import 'molecules/image-text-50-50.html' as image_text_50_50 %}
+{% import 'molecules/related-posts.html' as related_posts %}
```

Comment:
Can we group the organisms and molecules together like we do w/ the CSS to make it easier to find?